src/routes/webhooks/topgg.ts

```diff
@@ -0,0 +1,37 @@
+import ApiRequest from '@lib/structures/api/ApiRequest';
+import ApiResponse from '@lib/structures/api/ApiResponse';
+import { Route, RouteOptions } from 'klasa-dashboard-hooks';
+import { TOKENS } from '@root/config';
+import { UserSettings } from '@lib/types/settings/UserSettings';
+import { ApplyOptions } from '@skyra/decorators';
+
+@ApplyOptions<RouteOptions>({ name: 'webhooks/topgg', route: 'webhooks/topgg' })
+export default class extends Route {
+
+	public async post(request: ApiRequest, response: ApiResponse) {
+		if (request.headers.authorization !== TOKENS.WEBHOOK_B4D) return response.forbidden();
+		if (!request.body) return response.badRequest();
+
+		const body = request.body as Body;
+		try {
+			const user = await this.client.users.fetch(body.user);
+			const settings = await user.settings.sync();
```

Comment:
While unlikely, people can vote on bots that Skyra doesn't share a server with and so will have no data for. Will `user.settings.sync()` work if there was no entry for that user in the settings? I assume fetching will succeed since it fetches from the API but I'm not sure about this.

Reply:
It does succeed, if there's nothing in the DB, it does nothing but mark the entry as "doesn't exist in DB, execute CREATE instead of UPDATE in next save".